inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/user/UserServiceImpl.java

```diff
@@ -460,4 +466,28 @@ public void removeInChargeForConsume(String user, String operator) {
         }
     }
 
+    public void removeUserFromSession(Integer userId, String operator) {
```

Comment:
In the scenario of a single Manager service, this modification takes effect.
However, when multiple Manager services are deployed, this modification cannot take effect globally.

Approve first.